package.json

```diff
@@ -30,6 +30,9 @@
     "build:compile": "babel --presets es2015 build/hotshot.js -o build/hotshot.js",
     "build:minify": "uglifyjs build/hotshot.js -o build/hotshot.min.js -c -m",
 
+    "check": "npm run test",
```

Comment:
This feels kinda redundant but like you said it aligns with our current conventions so I guess its fine. 😄